### DISCUSSION TITLE
Great app, any chance of additional platforms?

### DISCUSSION THREAD

**Discussion by ozsmac** [audiamus/BookLibConnect#102] [Ideas]
This is an amazing app, thank you for the time and effort to let me listen to MY audible library in the app of my preference (Prologue on iOS).

My one hope is that one of the following platforms might be possible as a future supported target?

MacOS
  Windows 11 ARM
    Linux (Ubuntu or Fedora)

Also had anyone managed to get this running in Windows 11 arm in x86 emulation mode?

Understand this is likely not possible, but you never know :)

Why?
I've moved to MacOS for almost all of my computing at home and work, as a result I'm running an Intel NUC just to run BookLibConnect.

**Comment by audiamus** [comment]
> MacOS
Windows 11 ARM
Linux (Ubuntu or Fedora)

It's very unlikely. Not sure about Windows ARM, but the other platforms require a new GUI, based on XAML, and that would have to be Avalonia. However, the XAML GUIs, including Avalonia, have a different architecture and none of my current Windows Forms data flows in use can be migrated. Furthermore, while I frequently work with Linux systems (and running C# on them), the last piece of Apple hardware I touched was back in the 1980s.

**Comment by ozsmac** [comment]
Thank you for taking the time to reply @audiamus, really appreciate your hard work on with BookLibConnect.

I'll leave my fingers crossed that someday Windows ARM might be possible.  Good luck with the integration work with AAX Audio Converter & BookLibConnect.

**Reply by colvdv** [reply]
### [GUIDE] Book Lib Connect/AAX Audio Converter on Linux (WINE)

**Tested and working for Book Lib Connect v0.13.1 and AAX Audio Converter v1.18.2 on Fedora 41 using WINE 10.4.**

1. Download installers:
	- [Book Lib Connect v0.13.1](https://github.com/audiamus/BookLibConnect/releases/tag/v0.13.1)
	- [AAX Audio Converter v1.18.2](https://github.com/audiamus/AaxAudioConverter/releases/tag/v1.18.2)
	- [.NET SDK 6.0.428 Windows x64](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.428-windows-x64-installer)

2. Install using Wine:
	a. Open terminal
	b. cd to the download location: `cd ~/Downloads`
	c. Run `wine dotnet-sdk-6.0.428-win-x64.exe`, follow the prompts in the installer window.
	d. Run `wine BookLibConnect-0.13.1-Setup.exe`, follow the prompts and use the recommended settings.
	e. Run `wine AaxAudioConverter-1.18.2-Setup.exe`, follow the prompts and use the recommended settings.

5. Launch Book Lib Connect and follow the prompts to configure it. If met with any unhandled exceptions, just click continue until they go away. Do the same for AAX Audio Converter.

7. You should now be able to use Book Lib Connect and AAX Audio Converter on Linux!
